src/main/java/dk/cngroup/lentils/controller/ClientController.java

```diff
@@ -140,6 +141,10 @@ public String verifyCodeword(@PathVariable("id") final Long id,
             return CLIENT_VIEW_CYPHER_DETAIL;
         }
 
+        if (statusService.getCypherStatusByTeamAndCypher(team, cypher) != CypherStatus.PENDING) {
```

Comment:
Muzes tuto kontrolu pridat prosim i pro skipnuti sifry a pro ziskani hintu?

Reply:
pro skip je tady takovy kod:
```
        if (cypherStatus.equals(CypherStatus.PENDING)) {
            statusService.markCypher(cypher, user.getTeam(), CypherStatus.SKIPPED);
        }
```

Reply:
Tohle by chtelo napsat tak, aby na to sel napsat test.
Bud do sluzby, nebo otestovat kontroler :) Necham na zvazeni. Tohle me prijde jako celkem kriticka vec.

Reply:
pridano taky pro skipnuti a ziskani hintu (client)
napsany testy -> otazka: nemel bych ty testy nacpat vsechny do jedne Class? pouzivaji se tam casto stejne metody :) díky

Reply:
Přijde mě to, že tam bude moc testů, které nemají spolu nic moc společného. Spíš bych tam udělal abstraktní třídu, kde dáš ty společné věci a nechal ty podtřídy, kde budou specifické testy pro tu oblast, co testuješ. Takže třeba jak teď. Verify, Skip, Take hint.

Reply:
jedna abstraktni trida pro spolecne metody se udela v novem issue